suse_migration_services/units/setup_host_network.py

```diff
@@ -44,33 +44,27 @@ def main():
     log = logging.getLogger(Defaults.get_migration_log_name())
     root_path = Defaults.get_system_root_path()
 
+    system_mount = Fstab()
+    system_mount.read(
+        Defaults.get_system_mount_info_file()
+    )
+
     resolv_conf = os.sep.join(
         [root_path, 'etc', 'resolv.conf']
     )
     if not os.path.exists(resolv_conf):
         raise DistMigrationNameResolverException(
             'Could not find {0} on migration host'.format(resolv_conf)
         )
-    if has_host_resolv_setup(resolv_conf):
-        log.info('Copying {}'.format(resolv_conf))
-        shutil.copy(
-            resolv_conf, '/etc/resolv.conf'
-        )
-    else:
-        log.info('Empty {}, continuing without copying it'.format(resolv_conf))
-
-    sysconfig_network_providers = os.sep.join(
-        [root_path, 'etc', 'sysconfig', 'network', 'providers']
-    )
-    sysconfig_network_setup = os.sep.join(
-        [root_path, 'etc', 'sysconfig', 'network', '*']
-    )
```

Comment:
There's no need to move the variable assignment into the `try` block. These will not fail and from a code locality perspective it is only 1 line different, but it makes the `try` block significantly longer due to the formatting. As such distracting what the `try` block is actually supposed to catch.

Reply:
Thanks for catching this.  I adjusted